docs/script.rst

```diff
@@ -714,80 +705,78 @@ Table of optional parameters:
-
+.. note:: If the target file already exists, it will be replaced by the new one. Note also that, if the target is
+  a directory, the source file will be copied into that directory, maintaining the file's original name.
 
-When the source file is a directory, all the directory content is copied to the target folder. For example::
 
+When the source file is a directory, all its content is copied to the target directory::
 
   myDir = file('/some/path')
   myDir.copyTo('/some/new/path')
 
 
-If the target path does not exist, it will be created automatically.
+  If the target path does not exist, it will be created automatically.
 
-.. tip:: The ``copyTo`` method mimics the semantic of the Linux command ``cp -r <source> <target>``,
-  with the following caveats: Unix BASH distinguish between paths having or not having and ending slash, for example:
-  ``/some/path/name`` and ``/some/path/name/``. The first locates a regular file while the latter identifies a directory
-  location. With Nextflow, due to Java files API implementation, this is not possibles and both strings represents the same path.
-  If that path exists on the file systems it is handled accordingly (as a regular file or as a directory). If the path does not
-  exist, it is supposed to locate a regular file (and any parent directory will be created automatically).
+.. tip:: The ``copyTo`` method mimics the semantics of the Linux command ``cp -r <source> <target>``, with the
+         following caveat: While Linux tools often treat paths ending with a slash (e.g. ``/some/path/name/``)
+         as directories, and those not (e.g. ``/some/path/name``) as regular files, Nextflow (due to its use of
+         the Java files API) views both these paths as the same file system object. If the path exists, it is
+         handled according to its actual type (i.e. as a regular file or as a directory). If the path does not
+         exist, it is treated as a regular file, with any missing parent directories created automatically.
 
 
 
 Move files
 ----------
 
-You can move a file by using the method ``moveTo`` as shown in the example below::
+You can move a file by using the method ``moveTo``::
 
   myFile = file('/some/path/file.txt')
   myFile.moveTo('/another/path/new_file.txt')
 
 
-.. note:: When a file with the same name as the target already exists, it will be replaced by the new one. Note also that
-   when the target specifies a folder name instead of a file, the source file is moved in that folder maintaining the original name.
+.. note:: When a file with the same name as the target already exists, it will be replaced by the source. Note
+          also that, when the target is a directory, the file will be moved to (or within) that directory,
+          maintaining the file's original name.
 
-When the source file is a directory, all the directory content is moved to the new the destination folder::
```

Comment:
Suddenly the term _folder_ appears in place of _directory_. "Folder" to me is for a GUI context... I've changed "folder" to "directory" in several places to be more consistent with the rest of the document.